pkg/sql/plan/function/function_id.go

```diff
@@ -117,6 +117,7 @@ const (
 	HASH_AGG    // HASH_AGG
 	HEX_DECODE  // HEX_DECODE
 	HEX_ENCODE  // HEX_ENCODE
+	HEX         // HEX
```

Comment:
What happened to hex_encode and hex_decode?   What is the difference between these and hex?

Reply:
These two functions are 16 mechanism number encoding and decoding functions, which are predefined names as stations in plan2, but they have not been implemented！ @fengttt